.github/workflows/tests.yml

```diff
@@ -1,10 +1,14 @@
 name: 'Tests'
 
 on:
-  push:
-    branches:
-      - main
   pull_request:
+    paths-ignore:
+      - 'docs/**'
+      - '**/**.md'
```

Comment:
Could we also add `- '**/**.mdx'`?

Reply:
Of course, I actually updated it to include what you've requested by updating this to `**.mdx?`. You can get more info [here](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#filter-pattern-cheat-sheet)